dist/configs/config/server.cfg

```diff
@@ -66,6 +66,9 @@ set g_teamForceBalance 2
 //set g_doWarmup 1
 //set g_warmup 20
 
+// allow to fill teams with bots, can also be used in map rotation
```

Comment:
I think the map rotation suggestion is not good because the command will only run if the map is chosen via rotation, and not any other way. So you will have different settings depending whether the next map is picked automatically or by voting.